src/hex_tarball.erl

```diff
@@ -13,6 +13,33 @@
     {<<"Makefile">>, <<"make">>},
     {<<"Makefile.win">>, <<"make">>}
 ]).
+
+-define(DEPRECATED_META_FIELDS, [
+            <<"maintainers">>,
+            <<"contributors">>
+]).
+
+-define(META_FIELDS, [
+                       <<"name">>,
+                       <<"version">>,
+                       <<"files">>,
+                       <<"build_tool">>,
+                       <<"build_tools">>
+]).
+
+-define(REQUIRED_META_FIELDS, [
+                       <<"name">>,
+                       <<"version">>
```

Comment:
see above mentioned link for list of required fields, we're missing a few (e.g. licenses). That being said, perhaps it's better to focus on adding the simplest possible validations, make sure the code structure and API is good before making this complete - so I guess it's fine if this list is incomplete at this moment.

Reply:
See latest push.